addOns/grpc/src/main/javahelp/org/zaproxy/addon/grpc/resources/help/contents/variant.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+    <META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
+    <TITLE>gRPC Variant</TITLE>
+</HEAD>
+<BODY>
+<H1>gRPC Variant</H1>
+
+The gRPC variant allow injecting payloads in gRPC queries (Active Scan Input Vector support).
+
+<h2>Active Scan Input Vectors</h2>
+
+Values are injected for inline arguments in all queries proxied through ZAP.
+If a query uses variables, then variables are replaced inline and variable definitions are deleted.
```

Comment:
This needs to be updated, still using old text.

Reply:
The whole text, there are no "inline arguments" here nor "variables".

Reply:
is it good "Values are injected into all key-value pairs in all gRPC queries that are proxied through ZAP." ?

Reply:
queries -> values, but otherwise looks good